### ISSUE TITLE
Change the default beepSpeechModePitch value from 8000 Hz to 880 Hz

### ISSUE THREAD

**Issue by DrSooom** [nvaccess/nvda#9308]
### Steps to reproduce:
1. Press NVDA+S to switch from the speech on mode to the beep mode.
2. Do something. It doesn't matter.
### Actual behaviour:
Quote from issue #9206:
> • beepSpeechModePitch: text field (min = 55, max = 8372, default = 880)
Note: I guess the default value here is/was 8000 Hz, which hurts extremely in my ears. So reducing it directly to 880 Hz would be nice.
### Expected behaviour:
Change the default beepSpeechModePitch value from 8000 Hz to 880 Hz, which sounds more accurate and smooth.
#### NVDA installed/portable:
Both
#### NVDA version:
2018.4.1
### Related issues:
Issue #8711 and #9206

**Comment by josephsl** [comment]
Hi, you realize that this conflicts with top of screen beeps for mouse and 75 percent beep for progress bars, right? Thanks.

**Comment by DrSooom** [comment]
@josephsl: Yes, this is possible, but I guess the user will recognized the current context, where beeps occur, correctly. In the end he will get the opportunity to change this value via the NVDA Settings to the value he wants. See related issues.

btw: I also don't know, how many users are really using the beeps for the mouse movements. I know that this is a really old and nice feature for demonstrating, but how useful is it really in the daily work? (I don't want to say that you should put this out of NVDA. As long as the user is able to disable it, everything is still fine.)

**Comment by josephsl** [comment]
Hi, I use mouse beeps not only in mouse, but sometimes while using touchscreens. Thanks.

**Comment by DrSooom** [comment]
@josephsl: We should wait for more feedback from the community.

**Comment by josephsl** [comment]
Hi, that’s my intention at this time. Thanks.

**Comment by Brian1Gaff** [comment]
Yes, the problem is that 8k is getting close to older persons limit of 
hearing and the effect is to feel the sound rather than hearing it, as pain. 
I notice this at higher frequencies, say around 10khz. However not sure 
about the lower end. Could we not come up with a less strident sound, one 
that has softer starts and stops and sounds smoother?

Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
-----

**Comment by leonardder** [comment]
The default is actually 10000. I'm under the impression that the beep sounds at this rate contain some artifacts, probably because the audio quality can't deal with the overtones properly. I can already hear these around 3000 hz.

I really like setting it too 4400. like this on the python console

`import config; config.conf["speech"]["beepSpeechModePitch"]=4400`

**Comment by fernando-jose-silva** [comment]
I truly would like the nvda beeps to be lower, especially when using the one core synthesizers that have the lowest sound, which further causes a contrast between the volume of the beeps and the voice.
More like this call is to leave the most bass  beeps I believe that will already be a point of improvement.
So I vote to say the beeps in a more bass  tone.

**Comment by Brian1Gaff** [comment]
Yes it is a bit rough, I wonder what the waveform actually is?
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
----- Original Message -----

**Comment by DrSooom** [comment]
Suggestion to all users: Please change the setting first by your own and test it for a while.

1. Open the "nvda.ini" in Notepad++.
2. Search for "# Speech settings" followed by "[speech]" in a new line.
3. Add "beepSpeechModePitch = 880" below the line "[speech]"
4. Save the file and reload the configuration (NVDA+CTRL+R).

You can use this [notes table](https://danielmayr.at/abc/noten.html) to find out the correct Hz value for a note like 880 (Hz) for the note A5 (MIDI notation).

@leonardder: 4400, 8000 and 10000 Hz aren't valid notes. It is better to use a Hz value which is equal to a note, like 3520 Hz for A7. Note that the progress bar beep tones start at 110 Hz (parameter "beepMinHZ"), which is equal to A2. (In the hope that this value hasn't changed in the meantime too.)

**Comment by Mohamed00** [comment]
I think it's a triangle.

**Comment by man0528** [comment]
I like setting it too 4400.
